edb/schema/expraliases.py

```diff
@@ -79,12 +82,15 @@ def command_for_ast_node(
         modaliases = cls._modaliases_from_ast(schema, astnode, context)
         ctx = AliasCommandContext(
             schema,
-            op=sd._dummy_object,
+            op=sd._dummy_object,  # type: ignore
             scls=sd._dummy_object,
             modaliases=modaliases,
         )
 
+        mapping: Dict[Type[qlast.ObjectDDL], Any]
```

Comment:
Why `Any` and not `Type[sd.Command]`?

Reply:
mypy complains in all assignments below: the dictionary key is contravariant.
![image](https://user-images.githubusercontent.com/2576032/79806815-9a321680-8369-11ea-91de-8dd6bce28fdc.png)

I tried to use `TypeVar` to define a generic type, even using the `covariant=True` argument, but mypy is not happy anyway. In C#, this would be a case for a generic, because the key type argument for a Dictionary would be contravariant, too (e.g. no `Dictionary<IEnumerable, IEnumerable>` but rather `Dictionary<K, V> where V : IEnumerable`). I soon add a comment with the error when using a TypeVar.

Reply:
I deleted my previous comment (...). I got confused here, and missed the fact that I should have annotated each of the class attributes with `: Dict[Type[qlast.NamedDDL], Type[sd.Command]]`. :facepalm: 
Pushing a fix.